src/main/java/org/jitsi/videobridge/sctp/SctpManager.java

```diff
@@ -144,7 +164,13 @@ public int send(byte[] data, int offset, int length)
         {
             byte[] newBuf = ByteBufferPool.getBuffer(length);
             System.arraycopy(data, offset, newBuf, 0, length);
-            return innerSctpDataSender.send(newBuf, 0, length);
+
+            // Leave SCTP thread ASAP
+            TaskPools.IO_POOL.submit(() -> 
+            {
+                innerSctpDataSender.send(newBuf, 0, length);
```

Comment:
Delegating to just any IO thread breaks outgoing SCTP packet order when the machine is under load.

Reply:
The underlying transport (udp) doesn't guarantee order either

Reply:
You're right. Thanks for pointing out that we're not dealing with user message here, but rather with the underlying protocol. I missed that.